autonomi/README_WASM.md

```diff
@@ -1,95 +0,0 @@
-# Autonomi JS API
-
-Note: the JS API is experimental and will be subject to change.
```

Comment:
Are we sure we want to delete these WASM docs now?

Reply:
yes we gave up on wasm for now. I hope we introduce it at a later stage, but we need to ensure we do not hamper network or disk access when we do